setup.rst

```diff
@@ -125,7 +125,23 @@ On **Fedora** and other ``DNF`` based systems::
    $ sudo dnf builddep python3
 
 On **Debian**, **Ubuntu**, and other ``apt`` based systems, try to get the
-dependencies for the Python version that you're working on e.g.::
+dependencies for the Python using the ``apt``.
+
+First please make sure you have enabled the source packages in source list.
```

Comment:
I think removing "please" would be better, i.e: "First, make sure ..."
Additionally,  "source" -> "sources".